editor/src/components/canvas/controls/classname-select.tsx

```diff
@@ -491,10 +572,13 @@ export const ClassNameSelect: React.FunctionComponent = betterReactMemo('ClassNa
       }}
     >
       <WindowedSelect
+        menuIsOpen={true}
```

Comment:
is this added for debugging?

Reply:
Ha oops! Yeah I was using it to fix the styles :grinning: Will remove